spec/cypress/integration/login.spec.js

```diff
@@ -20,9 +20,9 @@ describe('Login', () => {
             full_name: fullName,
```

Comment:
A minor thing: Can we remove the `spec/cypress/videos/login.spec.js.mp4` video?

Reply:
I don't have it locally but I'll remove it outside of this branch.